INSTALLING.md

```diff
@@ -0,0 +1,63 @@
+# INSTALLING
+
+## A Note about Redis Persistence
+
+The application requires a Redis server that is configured with `Append-only file` persistence. This is so that application state can be maintained across Redis restarts. Notes on how to configure this are included below for each type of deployment.
+
+
+## Deploying using prebuilt Docker images
+
+### Standalone (complete) image
+
+The standalone container image is meant to be a complete solution for a provider. It contains a single instance of the main application, plus a pre-configured Redis server. Simply download the latest [container image](link) and deploy using your favorite container management system.
+
+The internal Redis server included in the complete image is already configured for persistence; it is simply necessary to configure your container pod to map the directory `/var/lib/redis` to a persistent storage volume.
+
+Follow the instructions below for [configuration](#configuration), with the exception that you should _not_ modify `REDIS_URL`, as it already points to the internal Redis server.
+
+### App-only image
+
+The app-only image is meant to be used for providers who would rather utilize a Redis instance in their own (or their cloud infrastructure provider's) external Redis instance or service. In this case, you need to ensure that the following settings are configured in your Redis instance:
+```
+appendonly true
+dir <base directory for Redis storage>
+appendonlydir <subdirectory of base directory where append-only persistence files are stored>
+```
+
+You must also minimally map `appendonlydir` (or the entire `dir`) to a persistent storage volume in your infrastructure environment
+
+## Building and Deploying the Application
+
+If you choose to build & deploy the application yourself, simply install the prerequisites:
+* NodeJS 18
+
+Note that, at present, due to limitaions in the `@dsnp/graph-sdk` module, the applcation can only be run on a `linux/amd64` platform. Support for other platforms is contingent on building & installing the `@dsnp/graph-sdk` module from source, which is outside the scope of this document.
+
+To build the application:
+```
+    npm run build
+```
+
+To run the application:
+```
+    npm start
+```
+
+## Configuration<a name="configuration"></a>
```

Comment:
The anchor is actually automatically added and you don't have to do the html anchor tag